websites/A/Aodesu/presence.ts

```diff
@@ -0,0 +1,122 @@
+const presence = new Presence({
+		clientId: "1101840416866844672",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+let iFrameVideo: boolean,
+	currentTime: number,
+	duration: number,
+	paused: boolean,
+	playback: boolean,
+	reproductor: string;
+
+presence.on(
+	"iFrameData",
+	(data: {
+		iframeVideo: {
+			duration: number;
+			iFrameVideo: boolean;
+			currTime: number;
+			paused: boolean;
+			reproductor: string;
+		};
+	}) => {
+		playback = data.iframeVideo.duration !== null ? true : false;
+		reproductor = data.iframeVideo.reproductor;
+		if (playback) {
+			({ iFrameVideo, paused, duration } = data.iframeVideo);
+			currentTime = data.iframeVideo.currTime;
+		}
+	}
+);
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+		largeImageKey: "https://i.imgur.com/CyVxys5.jpg",
+	};
+
+	presenceData.startTimestamp = browsingTimestamp;
+
+	if (document.location.pathname === "/") {
+		presenceData.smallImageKey = "home";
```

Comment:
Use Imgur URLs over Discord asset names. In addition, as you have many assets, put assets in an enum like so:
https://github.com/PreMiD/Presences/blob/3aa7e3fb8e54999012de1d1528f37bc907ae4213/websites/C/Crunchyroll/presence.ts#L52-L58